code/modules/mob/living/living.dm

```diff
@@ -1646,7 +1646,7 @@ Thanks.
 		item=what
 
 	if(!item)
-		return
+		return -1
```

Comment:
???

Reply:
I needed a return value that wasn't 0, so not having an item to throw could be distinguished from failing to throw an item. However, it also needed to not be 1, because I imagine that in the future someone may want to check the throw proc for success and in that case, the return values for succeeding to throw the item and not having an item to throw would need to be different.

I could not think of a better solution than returning -1 to signify succeeding to throw nothing.

Reply:
Cool, I suggest you leave a comment explaining this otherwise the next person to read this will also ask "???"

Reply:
I have added a comment explaining the return values next to the proc definition.

Reply:
Make it a return value instead of cryptic numbers. Even with the comment it's confusing.

Reply:
I'm not sure what you mean by "return value." Do you mean a string? Or a define?

Reply:
My fault, I didn't write "define" apparently.

Make it a define.